src/official-account/official-account.ts

```diff
@@ -252,6 +254,40 @@ class OfficialAccount extends EventEmitter {
     return ret.body
   }
 
+  async uploadTempFile (args: {
+    file: FileBox,
+    touser: string,
+    msgtype: OAMessageType
+  }): Promise<ErrorPayload> {
+    log.verbose('OfficialAccount', 'uploadTempFile(%s)', JSON.stringify(args))
+    const msgSet: Set<string> = new Set(['image', 'video', 'voice'])
+    if (!msgSet.has(args.msgtype)) {
+      throw Error(`msgtype <${args.msgtype}> is  not supported`)
+    }
+
+    const { buf, info } = await normalizeFileBox(args.file)
+    const mediaResponse = await this.simpleUnirest.post(`media/upload?access_token=${this.accessToken}&type=${args.msgtype}`).attach('attachments[]', buf, info)
+    const mediaPayload = JSON.parse(mediaResponse.body as string)
+
+    if (mediaPayload.errcode) {
+      throw Error('invalid media type')
+    }
+
+    const data = {
+      image:
+      {
+        media_id: mediaPayload.media_id,
+      },
+      msgtype: args.msgtype,
+      touser: args.touser,
+    }
+    const messageReq = await this.simpleUnirest
+      .post<ErrorPayload>(`message/custom/send?access_token=${this.accessToken}`)
+      .type('json')
+      .send(data)
+    return messageReq.body
```

Comment:
What's the purpose of returning `req.body` at here?

The typing of this function said that we should return an `ErrorPayload`, but your code below did not check any return type at all.

Please use void if you do not care the return value, or check your return value to make sure everything is expected.

Reply:
There is a payload from response, which contains `media_id` field. refer to : [upload temp file](https://developers.weixin.qq.com/doc/offiaccount/Asset_Management/New_temporary_materials.html) 

So, should we add a `MediaPayload` interface to define the return result of `sendFile` function ?